myuw/dao/calendar_mapping.py

```diff
@@ -12,40 +15,40 @@
 
 
 def get_calendars_for_current_user(request):
-    return _get_calendars(request, _get_enrollments(request))
+
+    try:
+
+        majors = get_majors(request)['rollup']
+        majors = [major.major_name for major in majors]
+
+        minors = get_minors(request)['rollup']
+        minors = [minor.short_name for minor in minors]
+
```

Comment:
Can we get {
        'majors': majors,
        'minors': minors
    }
in one call?